src/Test/Runner.elm

```diff
@@ -230,22 +236,62 @@ distributeSeeds runs seed test =
             }
 
         Internal.Labeled description subTest ->
-            let
-                next =
-                    distributeSeeds runs seed subTest
-            in
-            { seed = next.seed
-            , all = List.map (Labeled description) next.all
-            , only = List.map (Labeled description) next.only
-            , skipped = List.map (Labeled description) next.skipped
-            }
+            -- This fixes https://github.com/elm-community/elm-test/issues/192
+            -- The first time we hit a Labeled, we want to use the hash of
+            -- that label, along with the original seed, as our starting
+            -- point for distribution. Repeating this process more than
+            -- once would be a waste.
+            if hashed then
+                let
+                    next =
+                        distributeSeedsHelp True runs seed subTest
+                in
+                { seed = next.seed
+                , all = List.map (Labeled description) next.all
+                , only = List.map (Labeled description) next.only
+                , skipped = List.map (Labeled description) next.skipped
+                }
+            else
+                let
+                    intFromSeed =
+                        -- At this point, this seed will be the original
+                        -- one passed into distributeSeeds. We know this
+                        -- because the only other branch that does a
+                        -- Random.step on that seed is the Internal.Test
+                        -- branch, and you can't have a Labeled inside a
+                        -- Test, so that couldn't have come up yet.
+                        seed
+                            -- Convert the Seed back to an Int
+                            |> Random.step (Random.int 0 Random.maxInt)
+                            |> Tuple.first
+
+                    hashedSeed =
+                        -- Incorporate the originally passed-in seed
+                        (toString intFromSeed ++ description)
+                            -- Hash from String to Int
+                            |> FNV.hashString
+                            -- Convert Int back to Seed
+                            |> Random.initialSeed
```

Comment:
@Skinney I'd be curious if you have any thoughts on a better way to do this.

Goals:

1. Start with `intFromSeed : Int` and `description : String`
2. End up with a new `Int` that incorporates both of them
3. Have that resulting `Int` not follow any predictable patterns based on its two inputs (this isn't *critical* like in the crypto sense, but it needs to hold true most of the time)

I imagine there's some faster/better way to do this with integer arithmetic, but I was (perhaps unreasonably) worried about overflows and such.